src/Symfony/Component/Messenger/Transport/Serialization/SimpleSerializer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Serialization;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Encoder\XmlEncoder;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer as SymfonySerializer;
+
+class SimpleSerializer implements EncoderInterface, DecoderInterface
```

Comment:
given `Serializer` is the Symfony tailored implementation already, this could also be utilized into `Serializer::create()` with default encoders/normalizers for arguments.

Reply:
Yep, I definitely agree with this: it can be as simple as a public static factory method on the `Serializer` :)

Reply:
Done